app/models/post.rb

```diff
@@ -3,14 +3,8 @@ class Post < ApplicationRecord
   has_many :likes
   belongs_to :author, class_name: 'User'
 
-  validates :title, presence: true, length: { maximum: 250 }
-  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
```

Comment:
- [ ] Kindly make sure that the `Post` model has validations, for title comments_counter, and likes_conter, your validations got removed when you revert the commit